### ISSUE TITLE
Pre-work Checklist: Developer: Reid Hommedahl

### ISSUE THREAD

**Issue by rdhmdhl** [hackforla/website#4623]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by rdhmdhl** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

I've spent around ~7 hours including the initial onboarding which was about 3 hours, and the Sunday team meeting which was about 1.5 hours. About 2.5 hours of reading the contributing doc, setting up the dev environment, and making pull requests.

**Comment by rdhmdhl** [comment]
Update: completed two "first good issues".
https://github.com/hackforla/website/issues/4605
https://github.com/hackforla/website/issues/4483

**Comment by rdhmdhl** [comment]
Update: completed medium issue
#4050

**Comment by rdhmdhl** [comment]
Update: completed small issue on 5/25/23
#4706

**Comment by ExperimentsInHonesty** [comment]
@rdhmdhl I am closing this prework issue since you have done all the steps now.  The last step was doing a update on your issue even when you don't have any blockers or questions, and you have done that on this issue https://github.com/hackforla/website/issues/4050

**Comment by HackforLABot** [comment]
## Activity Log: rdhmdhl
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- rdhmdhl opened: https://github.com/hackforla/website/issues/4623 at 05/08/2023, 08:30 PM PDT
- rdhmdhl assigned: https://github.com/hackforla/website/issues/4623 at 05/08/2023, 08:31 PM PDT
- rdhmdhl assigned: https://github.com/hackforla/website/issues/4483 at 05/09/2023, 09:34 PM PDT
- rdhmdhl opened: https://github.com/hackforla/website/pull/4645 at 05/09/2023, 09:50 PM PDT
- rdhmdhl assigned: https://github.com/hackforla/website/issues/4485 at 05/10/2023, 08:52 PM PDT
- rdhmdhl closed: https://github.com/hackforla/website/pull/4645#event-9217702576 at 05/10/2023, 09:14 PM PDT
- rdhmdhl closed as completed: https://github.com/hackforla/website/issues/4485#event-9217884689 at 05/10/2023, 10:00 PM PDT
- rdhmdhl opened: https://github.com/hackforla/website/pull/4651 at 05/10/2023, 10:07 PM PDT
- rdhmdhl submitted review: https://github.com/hackforla/website/pull/4666#pullrequestreview-1425594000 at 05/14/2023, 12:46 PM PDT
- rdhmdhl submitted review: https://github.com/hackforla/website/pull/4656#pullrequestreview-1425595989 at 05/14/2023, 01:09 PM PDT
- rdhmdhl assigned: https://github.com/hackforla/website/issues/4605 at 05/14/2023, 03:53 PM PDT
- rdhmdhl opened: https://github.com/hackforla/website/pull/4667 at 05/14/2023, 04:20 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4623#issuecomment-1547026761 at 05/14/2023, 04:27 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4605#issuecomment-1547027773 at 05/14/2023, 04:31 PM PDT
- rdhmdhl unassigned: https://github.com/hackforla/website/issues/4485 at 05/16/2023, 09:30 AM PDT
- rdhmdhl PR was merged: https://github.com/hackforla/website/pull/4667#event-9263819062 at 05/16/2023, 06:47 PM PDT
- rdhmdhl issue closed by PR 4667: https://github.com/hackforla/website/issues/4605#event-9263819154 at 05/16/2023, 06:47 PM PDT
- rdhmdhl PR was closed: https://github.com/hackforla/website/pull/4651#event-9263833874 at 05/16/2023, 06:50 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4483#issuecomment-1550668976 at 05/16/2023, 09:06 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4483#issuecomment-1551781555 at 05/17/2023, 10:14 AM PDT
- rdhmdhl opened: https://github.com/hackforla/website/pull/4695 at 05/18/2023, 05:48 PM PDT
- rdhmdhl opened: https://github.com/hackforla/website/pull/4696 at 05/18/2023, 05:53 PM PDT
- rdhmdhl PR was closed: https://github.com/hackforla/website/pull/4696#event-9284802954 at 05/18/2023, 05:54 PM PDT
- rdhmdhl opened: https://github.com/hackforla/website/pull/4697 at 05/18/2023, 06:23 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4483#issuecomment-1553883870 at 05/18/2023, 06:24 PM PDT
- rdhmdhl PR was merged: https://github.com/hackforla/website/pull/4697#event-9296192877 at 05/21/2023, 08:44 AM PDT
- rdhmdhl issue closed by PR 4697: https://github.com/hackforla/website/issues/4483#event-9296192907 at 05/21/2023, 08:44 AM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4623#issuecomment-1556236134 at 05/21/2023, 10:37 AM PDT
- rdhmdhl assigned: https://github.com/hackforla/website/issues/4706 at 05/23/2023, 06:33 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4706#issuecomment-1560337842 at 05/23/2023, 06:34 PM PDT
- rdhmdhl opened: https://github.com/hackforla/website/pull/4710 at 05/23/2023, 07:14 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/pull/4710#issuecomment-1560394505 at 05/23/2023, 08:02 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/pull/4710#issuecomment-1562239299 at 05/24/2023, 09:17 PM PDT
- rdhmdhl PR was merged: https://github.com/hackforla/website/pull/4710#event-9343295884 at 05/25/2023, 11:34 AM PDT
- rdhmdhl issue closed by PR 4710: https://github.com/hackforla/website/issues/4706#event-9343296569 at 05/25/2023, 11:34 AM PDT
- rdhmdhl assigned: https://github.com/hackforla/website/issues/4050#issuecomment-1547138979 at 05/25/2023, 09:26 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4050#issuecomment-1563794590 at 05/25/2023, 09:29 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/pull/4731#issuecomment-1567426090 at 05/29/2023, 11:55 AM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/pull/4746#issuecomment-1567426809 at 05/29/2023, 11:56 AM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4050#issuecomment-1567533798 at 05/29/2023, 02:36 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4050#issuecomment-1567539390 at 05/29/2023, 02:48 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4050#issuecomment-1569296229 at 05/30/2023, 04:59 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/pull/4746#issuecomment-1569362340 at 05/30/2023, 06:35 PM PDT
- rdhmdhl opened: https://github.com/hackforla/website/issues/4766 at 06/01/2023, 01:22 AM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4050#issuecomment-1586255961 at 06/11/2023, 10:26 AM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/pull/4796#issuecomment-1590262840 at 06/13/2023, 05:51 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4050#issuecomment-1608729725 at 06/26/2023, 08:37 PM PDT
- rdhmdhl submitted review: https://github.com/hackforla/website/pull/4731#pullrequestreview-1502160893 at 06/27/2023, 07:43 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4050#issuecomment-1627790646 at 07/09/2023, 11:29 AM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/pull/4926#issuecomment-1631753931 at 07/11/2023, 07:26 PM PDT
- rdhmdhl submitted review: https://github.com/hackforla/website/pull/4731#pullrequestreview-1527330233 at 07/12/2023, 03:42 PM PDT
- rdhmdhl submitted review: https://github.com/hackforla/website/pull/4926#pullrequestreview-1527428808 at 07/12/2023, 06:17 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4050#issuecomment-1641202769 at 07/18/2023, 05:38 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4050#issuecomment-1646282976 at 07/21/2023, 03:08 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4050#issuecomment-1650930065 at 07/25/2023, 08:40 PM PDT
- rdhmdhl issue closed as completed: https://github.com/hackforla/website/issues/4050#event-9973475211 at 08/01/2023, 01:31 AM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4623#issuecomment-1664736096 at 08/03/2023, 03:52 PM PDT
- rdhmdhl commented on: https://github.com/hackforla/website/issues/4623#issuecomment-1664737655 at 08/03/2023, 03:55 PM PDT
- rdhmdhl opened: https://github.com/hackforla/website/issues/5205 at 08/14/2023, 07:01 PM PDT
- rdhmdhl assigned: https://github.com/hackforla/website/issues/5205 at 08/14/2023, 07:01 PM PDT
- rdhmdhl issue closed as completed: https://github.com/hackforla/website/issues/4623#event-10584530548 at 10/08/2023, 09:22 AM PDT
- rdhmdhl unassigned: https://github.com/hackforla/website/issues/5205 at 01/26/2024, 01:35 PM PST
